lib/vsc/filesystem/quota/entities.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: latin-1 -*-
```

Comment:
a whole new module without single unittest?

Reply:
I'll bring over the tests as well, but this is deprecated for us, it's only still needed for VUB. But you're right. my bad.